server/spec/test_integration_WAPI.rb

```diff
@@ -90,7 +97,7 @@ def test_outside_exception_passed_on
     # check that unknown errors are passed on.
     r = w.get_request("/Students/#{@uniqname}/Terms.XXX")
     logger.debug "#{__LINE__}: toepo: r "+r.inspect
-    assert_equal 666, r.meta_status, "missed capturing exception"
+    assert_equal 400, r.meta_status, "missed capturing exception: status returned: #{r.meta_status}"
```

Comment:
The API changed so that the 400 is now the expected value for this test.  666 is still used for errors we don't explicitly deal with.